Apps/WebClient/src/ClientApp/test/timeline.test.ts

```diff
@@ -39,7 +39,7 @@ function createWrapper(options?: GatewayStoreOptions): Wrapper<TimelineView> {
     });
 }
 
-describe("Timeline view", () => {
+describe("Report View", () => {
```

Comment:
This seems like an unintentional change which should be reverted.

Reply:
good catch. Its fixed now